.ci/mellanox/ompi_test.sh

```diff
@@ -0,0 +1,905 @@
+#!/bin/bash -eEl
```

Comment:
Is it a copy of [test_jenkins.sh](https://github.com/mellanox-hpc/jenkins_scripts/blob/master/jenkins/ompi/ompi_jenkins.sh) from https://github.com/mellanox-hpc/jenkins_scripts? Any changes you've made?

Reply:
Yes, it's changed partially. Finally it's decided to keep it separately from the OpenMPI repo - PR is to be updated.